docs/index.rst

```diff
@@ -1,27 +1,9 @@
-sqlalchemy-redshift
-===================
+===============================================
+Welcome to sqlalchemy-redshift's documentation!
+===============================================
```

Comment:
This is a bit too much personality for my taste. I like a header to just name the project and not draw attention to itself.